.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Test (Liquidsoap)
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    types: [opened, synchronize]
+    paths: ["**/*.liq"]
+
+jobs:
+  liquidsoap-test-transcoder-stereo:
+    name: Liquidsoap Test Transcoder Stereo
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
We may need to enable renovate on this

Reply:
not need, Renovate already know how to handle this syntaxt,

see this example on dib project : https://github.com/radiofrance/dib/blob/main/.github/workflows/qa.yml#L23